portal/views/assessment_engine.py

```diff
@@ -814,11 +811,13 @@ def assessment_update(patient_id):
 
     try:
         QuestionnaireResponse.validate_document(updated_qnr)
-    except jsonschema.ValidationError as e:
+        QuestionnaireResponse.validate_authored(
+            FHIR_datetime.parse(updated_qnr.get('authored')))
+    except (jsonschema.ValidationError, NoFutureDates) as e:
         return jsonify({
             'ok': False,
-            'message': e.message,
-            'reference': e.schema,
+            'message': str(e),
```

Comment:
Surprised you had to cast this now, if not before...

Reply:
@ivan-c  Correct - this was an overlooked python2-3 required change.  Apparently this catch block hasn't been hit.?!